src/scheduler/batch/main.ts

```diff
@@ -62,9 +62,9 @@ export async function populateCurrentAnnualReview(lists: List[]): Promise<void>
 }
 
 export async function updateListsForAnnualReview(today: Date): Promise<void> {
-  const annualReviewStartDate = addDays(today, schedulerMilestoneDays.post.ONE_MONTH);
+  const annualReviewStartDate = subDays(today, schedulerMilestoneDays.post.ONE_MONTH);
```

Comment:
I think this needs to be reverted per the comment left on [lists.ts](https://github.com/UKForeignOffice/lists/pull/601/files#diff-31bd9ddf9173dc213fdf3d5002a79e8fc066adc3ec483ca51f256c7cbdb50544)